app/helpers/likes_helper.rb

```diff
@@ -0,0 +1,2 @@
+module LikesHelper
+end
```

Comment:
if is empty, why add this?

Reply:
I will delete the file because it is not used.